packages/components/src/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -52,7 +75,12 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<Props & Injected
         nameExpressionInfoUrl: '',
         nameExpressionPlaceholder: 'S-\${now:date}-\${dailySampleCount}',
         appPropertiesOnly: false,
-        helpTopic: DEFINE_SAMPLE_TYPE_TOPIC
+        helpTopic: DEFINE_SAMPLE_TYPE_TOPIC,
+        sampleAliasCaption: 'Sample Alias',
+        sampleTypeCaption: 'Sample Type',
+        dataClassAliasCaption: 'DataClass Alias',
+        dataClassTypeCaption: 'Dataclass',
+        dataClassParentageLabel: 'dataclass'
```

Comment:
For the ones that are visible to users, we should use spaces (e.g., 'Data Class')